.github/workflows/test.yml

```
@@ -140,4 +140,15 @@ jobs:
     - run: docker ps  # show the services against which we'll be testing
     - run: npm install
     - run: npm ls --all || true
-    - run: npm test
+    - name: npm test
+      run: npm test
+    - name: debug test_output
+      run: ls test_output/*.* || true
+    - uses: v1v/otel-upload-test-artifact-action@v2
+      if: always()
+      with:
```

Should we continue if something bad happened when generating the test traces?

```suggestion
      continue-on-error: true
      with:
```

---

I would presume that we would never want a telemetry error to cause a PR not to pass a GH check.